common/test/acceptance/pages/studio/import_export.py

```diff
@@ -294,3 +318,26 @@ class ImportLibraryPage(ImportMixin, TemplateCheckMixin, LibraryLoader, LibraryP
     """
     Import page for Libraries
     """
+    def get_nav_help_css(self):
```

Comment:
@Muddasser Why we have same code(methods) in `ImportLibraryPage` and `ExportLibraryPage`?